mojave/items/dogtags.dm

```diff
@@ -0,0 +1,102 @@
+/* The old dogtags
+* and medallions from BD.
+* all have the dogtag subtype
+*/
+
+/obj/item/card/id/ms13
+	name = "\improper ID tag"
+	desc = "A simple identification tag. This is a base class and you shouldn't be seeing it."
+	icon = 'mojave/icons/objects/dogtags.dmi'
+	icon_state = "holodogtag"
+	uses_overlays = FALSE
+	var/datum/bank_account = null
+
+/obj/item/card/id/ms13/bos
+	name = "holo dogtag"
+	desc = "An advanced holographic dogtag, that shows the duty of a BoS member."
+	icon_state = "holodogtag"
+
+/obj/item/card/id/ms13/deputy
+	name = "deputy's badge"
+	desc = "A silver badge which shows honour and dedication."
+	assignment = "Deputy"
+	icon_state = "deputy"
+
+/obj/item/card/id/ms13/deputy/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/card/id/ms13/sheriff))
+		registered_name = stripped_input(user, "Who do you want to designate as your deputy?", , "", MAX_NAME_LEN)
+		to_chat(user, "You scribble the [registered_name] for the name on the badge.")
+		update_label()
+	return ..()
+
+/obj/item/card/id/ms13/sheriff
+	name = "sheriff's badge"
+	desc = "A golden Sheriff's badge."
+	icon_state = "sheriff"
+
+/obj/item/card/id/ms13/ncrrecruit
+	name = "recruit's tags"
+	desc = "A dog tag proving enlistment in the NCR, issued to local recruits."
+	assignment = "NCR Recruit"
+	icon_state = "ncrdogtagtrooper"
+
+/obj/item/card/id/ms13/ncrrecruit/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/card/id/ms13/ncrlieutenant))
+		registered_name = stripped_input(user, "Who do you want to designate as a recruit?", , "", MAX_NAME_LEN)
+		to_chat(user, "You scribble the [registered_name] for the name on the dogtag.")
+		update_label()
+	return ..()
+
+/obj/item/card/id/ms13/ncrtrooper
```

Comment:
Dogtags get the in-game name of ID cards for some reason. Not a super pressing bug that will hold up a merge, but worth noting here.

Reply:
I suspect they're getting a name from some base code proc.